src/marten/cli/manage/command/clearsessions.cr

```diff
@@ -0,0 +1,41 @@
+module Marten
+  module CLI
+    class Manage
+      module Command
+        class ClearSessions < Base
+          command_name :clearsessions
+          help "Clears all expired sessions."
+
+          @no_prompt : Bool = false
+
+          def setup
+            on_option("-y", "Do not show prompts to the user") { @no_prompt = true }
+          end
+
+          def run
+            unless no_prompt?
+              print("All expired sessions will be removed.")
+              print("These sessions can't be restored.")
+              print("Do you want to continue [yes/no]?", ending: " ")
+              unless %w(y yes).includes?(stdin.gets.to_s.downcase)
+                print("Cancelling...")
+                return
+              end
+            end
+
+            print(style("Clearing expired sessions", fore: :light_blue, mode: :bold), ending: "\n")
+
+            HTTP::Session::Store.registry.each_value do |session_store_klass|
+              store = session_store_klass.new(nil)
+              store.clear_expired_entries
+            end
```

Comment:
Technically we should only clear expired entries for the configured sessions store. For example you could ensure this is the case by doing something like that:

```suggestion
            session_store_klass = HTTP::Session::Store.get(Marten.settings.sessions.store)
            session_store_klass.new(nil).clear_expired_entries
```

That `#clear_expired_entries` will only get called on the cookie store if this is the one being used by the current Marten project. 🙂